bin/gatsby.js

```diff
@@ -30,33 +31,53 @@ var loadGatsby = function (path) {
   require(path)
 }
 
-var loadGlobalGatsby = function () {
-  var commandsToIgnore = ['new', '--help']
-  if (commandsToIgnore.indexOf(process.argv[2]) === -1) {
-    console.log(
+/*
+  These commands can:
+  - Use a global install as a fallback if no local install is present
+  - Bypass the required file check
+ */
+const commandsToIgnore = [undefined, 'new', '--help', '-h', '--version', '-V']
+const isIgnored = commandsToIgnore.includes(process.argv[2])
+
+const requiredFiles = [
+  'html.js',
```

Comment:
Perhaps if we could modify the required file checker to support glob syntax then we could just do `'html.js*'`